nsxt/resource_nsxt_policy_host_transport_node_collection.go

```diff
@@ -0,0 +1,277 @@
+/* Copyright © 2023 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package nsxt
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/vmware/vsphere-automation-sdk-go/runtime/protocol/client"
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/infra"
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/infra/sites/enforcement_points"
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/model"
+)
+
+func resourceNsxtPolicyHostTransportNodeCollection() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceNsxtPolicyHostTransportNodeCollectionCreate,
+		Read:   resourceNsxtPolicyHostTransportNodeCollectionRead,
+		Update: resourceNsxtPolicyHostTransportNodeCollectionUpdate,
+		Delete: resourceNsxtPolicyHostTransportNodeCollectionDelete,
+		Importer: &schema.ResourceImporter{
+			State: nsxtPolicyPathResourceImporter,
```

Comment:
I don't think this can work out of the box here, since enforcement point and site need to be set before Read is called?